custom-graders/v2/README.md

```diff
@@ -0,0 +1,37 @@
+# New V2 Format Custom grader samples
+
```

Comment:
Add 
"We are upgrading the infrastructure system for Coursera Programming Assignment auto-graders. This repository has now been updated with "v2" format autograders in the custom-graders/v2 folders. This "v2" format ensures that your autograder will be compatible with our new infrastructure."